src/VisualStudio/Core/Def/ValueTracking/ValueTrackingTreeItemViewModel.cs

```diff
@@ -42,34 +53,55 @@ public IList<Inline> Inlines
         {
             get
             {
+                if (ClassifiedSpans.IsDefaultOrEmpty)
+                {
+                    return new List<Inline>();
+                }
+
                 var classifiedTexts = ClassifiedSpans.SelectAsArray(
                    cs => new ClassifiedText(cs.ClassificationType, _sourceText.ToString(cs.TextSpan)));
 
+                var spanStartPosition = TextSpan.Start - ClassifiedSpans[0].TextSpan.Start;
+                var spanEndPosition = TextSpan.End - ClassifiedSpans[0].TextSpan.End;
+
                 return classifiedTexts.ToInlines(
-                    _classificationFormatMap,
-                    _classificationTypeMap);
+                    TreeViewModel.ClassificationFormatMap,
+                    TreeViewModel.ClassificationTypeMap,
+                    (run, classifiedText, position) =>
+                    {
+                        if (TreeViewModel.HighlightBrush is not null)
```

Comment:
when would tihs be null?

Reply:
If it's not set in the properties, same as https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/Core/Def/Implementation/FindReferences/Entries/DocumentSpanEntry.cs#L94